data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/mlcp-flow-transform.xqy

```diff
@@ -0,0 +1,32 @@
+xquery version "1.0-ml";
+
+module namespace mlcpFlow = "http://marklogic.com/hub-in-a-box/mlcp-flow-transform";
+
+import module namespace flowLib = "http://marklogic.com/hub-in-a-box/flow-lib" at "/com.marklogic.hub/lib/flow-lib.xqy";
+
+declare function mlcpFlow:transform(
+  $content as map:map,
+  $context as map:map
+) as map:map*
+{
+  let $uri := map:get($content, "uri")
+  let $_ := xdmp:log('mlcp-flow-transform received: ' || $uri)
+  
+  let $paramNodes := xdmp:unquote(map:get($context, 'transform_param'))/node()/*
```

Comment:
This is a lot of work to get the transform_param into a map. If you pass it in as json instead you can do this:

```xquery
xdmp:from-json-string(map:get($context, 'transform_param'))
```

That should give you a map